network.ts

```diff
@@ -66,3 +66,26 @@ export const baseURIs: TablelandNetworkConfig = {
   localhost: localTablelandURI,
   "local-tableland": localTablelandURI,
 };
+
+// See validator for config for more details; these times consider the `NewBlockPollFreq` & `MinBlockDepth` values
+// Mainnets: https://github.com/tablelandnetwork/go-tableland/blob/main/docker/deployed/mainnet/api/config.json
+// Testnets: https://github.com/tablelandnetwork/go-tableland/blob/main/docker/deployed/testnet/api/config.json
+export const blockPollingTimes: TablelandNetworkConfig = {
```

Comment:
@sanderpick for the [`pollingController` PR](https://github.com/tablelandnetwork/tableland-js/pull/36#issuecomment-1773412643), i've added block times to `networks.ts` so that the SDK can use them.

i pulled these values from the `go-tableland` repo. e.g., if a chain has `NewBlockPollFreq` of `10s` and `MinBlockDepth` of `1`, the value used below would be `20_000`. or for OP and Arbitrum, we have 5s polling and zero block depth, so i'm using `5_000` as the value.

but, now that i think about it...in reality, the polling frequencies from `go-tableland` aren't _actually_ the block times, so i guess these values could be misleading. e.g., FIL is 30 second blocks with 5 block depth. instead, maybe the polling periods below might want to be: 
```
b * (d + 1) + b
```

where `b` is block time and `d` is depth. we need `d+1` to consider the block the tx is mined in, and the addition of `b` at the end is a margin of safety. thus, FIL would have a value of `210_000` (30*6+30). ethereum would be `40_000`. matic would be `15_000`. all other L2s would be `10_000`.

anyways, i figure the usage with the polling controller would be something like:
```js
import { helpers } from "@tableland/sdk";
// This import would be done under the hood within the SDK so that chains have default times, but shown here an as example
import { blockPollingTimes } from "@tableland/evm"; 

// For Filecoin testnet
const controller = helpers.createPollingController(blockPollingTimes["filecoin-calibraion"], 1500); // polling timeout and interval
```

Reply:
note: i ended up adjusting things with the logic defined above.

Reply:
Making sure I understand here.  These values are the maximum amount of time a client should poll the validator for a table txn receipt?
If so, what if we use the name `validatorPollingTimeouts`?

Reply:
@joewagner i like that better! updated.

Reply:
in the past I've used [10m timeout for filecoin](https://github.com/tablelandnetwork/evm-tableland/blob/main/scripts/deploy.ts#L7). not sure where that came from exactly, but I think it was the "finality" that we are considering in the validator. maybe i was overestimating. 

anyway, this looks good to me!

Reply:
(i'll keep an eye on if anyone mentions FIL issues in the future wrt the SDK abort signal fix...at least for filecoin calibration, it was working w/ these timeouts. can always increase as needed.)